### PR TITLE
test: take a screenshot upon failure example

### PR DESCRIPTION
**Pull request by pavelfeldman**
drive-by timeout for test only.

### CODE REVIEW

File: test/runner/fixtures.js [hunk 1]
```diff
@@ -104,31 +104,47 @@ class FixturePool {
...
+  wrapTestCallback(callback, timeout, info) {
     if (!callback)
       return callback;
+    const testResult = { success: true };
     return async() => {
       try {
-        return await this.resolveParametersAndRun(callback);
+        await this.resolveParametersAndRun(callback, timeout);
+      } catch (e) {
+        testResult.info = info;
```
Comment (dgozman) [inline test/runner/fixtures.js:141]: Let's fill these for success as well.

File: test/test-runner.spec.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import './test-runner-helper';
+import { registerFixture } from './runner';
+import { convertSelectOptionValues } from '../lib/rpc/client/elementHandle';
```
Comment (dgozman) [inline test/test-runner.spec.ts:19]: Unused?

**Review by dgozman — APPROVED** [review]
Wonderful!

**Review by JoelEinbinder — APPROVED** [review]

File: test/runner/fixtures.js (hunk 1 above)
Comment (pavelfeldman) [reply test/runner/fixtures.js:141]: Done

**Pull request merged**